en/guide/missions.md

```diff
@@ -126,18 +128,37 @@ std::shared_ptr<MissionItem> add_mission_item(double latitude_deg,
     new_item->set_camera_action(camera_action);
     return new_item;
 }
-    
-    
+
+
 mission_items.push_back(
-    add_mission_item(47.398170327054473,
-                     8.5456490218639658,
-                     10.0f, 5.0f, false,
-                     20.0f, 60.0f,
-                     MissionItem::CameraAction::NONE));
+    make_mission_item(47.398170327054473,
+                      8.5456490218639658,
+                      10.0f, 5.0f, false,
+                      20.0f, 60.0f,
+                      MissionItem::CameraAction::NONE));
 ```
 
+## Import a Mission from a QGC Plan {#import_qgc_plan}
+
+DroneCore allows you to import a mission from a *QGroundControl* plan (the imported mission can then be uploaded to a vehicle).
+
+> **Note** To export a mission plan from the *QGroundControl* use the [Sync Tool](https://docs.qgroundcontrol.com/en/PlanView/PlanView.html#sync) (**Plan View > Sync Tool**, and then select **Save to File**).
+
+The mission is imported using the static [import_qgroundcontrol_mission](../api_reference/classdronecore_1_1_mission.md#classdronecore_1_1_mission_1a7c73e97e5c1395a7451bb659d03e5f57) method. The method will fail with an error if the plan file cannot be found, cannot be parsed, or if it contains mission items that are [not supported](#supported_mission_commands).

```

Comment:
>The method will fail with an error if the plan file cannot be found, cannot be parsed, or if it contains mission items that are [not supported](#supported_mission_commands).

The method will fail with an error if plan file cannot be found, **or** it cannot be parsed, or if it contains mission items that are [not supported](#supported_mission_commands).
(Added *or* )

Reply:
My way is correct (so is yours, but I only change things if the alternative is "better").